run.linkerd.io/public/index.html

```diff
@@ -1 +1,12 @@
-<!DOCTYPE html><html><head><title>https://linkerd.io/</title><link rel="canonical" href="https://linkerd.io/2.10/getting-started/" /><meta name="robots" content="noindex"><meta http-equiv="content-type" content="text/html; charset=utf-8" /><meta http-equiv="refresh" content="0; url=https://linkerd.io/2.10/getting-started/" /></head></html>
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>https://linkerd.io/</title>
+    <link rel="canonical" href="https://linkerd.io/2.11/getting-started/" />
```

Comment:
Actually let's make this just `2`, which cloudflare automatically redirects to whatever the latest version is
```suggestion
    <link rel="canonical" href="https://linkerd.io/2/getting-started/" />
```